server/src/json-api-deserialize.pipe.ts

```diff
@@ -33,6 +33,9 @@ export class JsonApiDeserializePipe implements PipeTransform {
         contacts: {
           valueForRelationship: relationship => relationship.id,
         },
+        'related-actions': {
+          valueForRelationship: relationship => relationship.id,
+        },
       }).deserialize(value);
```

Comment:
I hope this wasn't as big a headache for as it was last time

Reply:
not at all :)